src/theory/strings/theory_strings_utils.h

```diff
@@ -0,0 +1,43 @@
+/*********************                                                        */
+/*! \file theory_strings_utils.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Andrew Reynolds
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Util functions for theory strings.
+ **
+ ** Util functions for theory strings.
+ **/
+
+#include "cvc4_private.h"
+
+#pragma once
+
+#include <set>
+#include <sstream>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+#include "expr/node_manager.h"
+
+namespace CVC4 {
+namespace theory {
+namespace strings {
+namespace utils {
+
+/**
+ * Make the conjunction of nodes in a. Removes duplicate conjuncts, returns
+ * true if a is empty, and a single literal if a has size 1.
+ */
+Node mkAnd(std::vector<Node>& a);
```

Comment:
I think this is a bigger discussion, that we've debated a lot in the past and won't be easy to quickly resolve. I'd like to revisit this but its out of scope for the PR.

This PR is regardless progress towards this since it is making strings/BV more uniform as they both will now have a utils file.

Reply:
I tend to agree with this and yes, we should discuss this again. To have a utils file is at least a step forward :)